draft-ietf-httpbis-header-compression.xml

```diff
@@ -1227,24 +1227,38 @@ else
                         might occur for shorter values more quickly or with
                         higher probability than for longer values.
                     </t>
-                    <t>
-                        Implementations might also choose to protect certain
-                        header fields that are known to be highly valued, such
-                        as the Authorization or Cookie header fields, by
-                        disabling or further limiting compression.
-                    </t>
                 </section>
 
                 <section title="Never Indexed Literals">
                     <t>
+                        Implementations can also choose to protect sensitive
+                        header fields by not compressing them and instead
+                        encoding their value as literals.
+                    </t>
+                    <t>
                         Refusing to generate an indexed representation for a
                         header field is only effective if compression is avoided
                         on all hops.  The never indexed literal (see <xref
                             target="literal.header.never.indexed"/>) can be used
                         to signal to intermediaries that a particular value was
-                        intentionally sent as a literal.  An intermediary MUST
-                        NOT re-encode a value that uses the never indexed
-                        literal with a representation that would index it.
+                        intentionally sent as a literal.
+                    </t>
+                    <t>
+                        An intermediary MUST NOT re-encode a value that uses the
+                        never indexed literal representation with another
+                        representation that would index it. If HPACK is used 
+                        for re-encoding, the never indexed literal
+                        representation MUST be used.
+                    </t>
+                    <t>
+                        The decision on whether a header field is sensitive or
+                        not is highly dependent on the context. As a generic
+                        guidance, header fields used for conveying highly valued
+                        information, such as the Authorization or Cookie header
+                        fields, can be considered to be on the more sensitive
+                        side.  In addition, a header field with a short value
+                        has potentially a smaller entropy and can be more at
+                        risk.
```

Comment:
How about:
> The choice to use a never indexed literal representation for a header field depends on several factors.  Since HPACK compresses based on the entirety of a value, short or low-entropy values are more readily recovered by an adversary.  Therefore, an encoder might choose to disable compression for values with low entropy.  Encoders might also choose to disable compression for header fields that are considered to be highly valuable or sensitive to recovery, such as the Cookie or Authorization header fields.

> An encoder might prefer compression for values that have little or no value if they were exposed.  For instance, a User-Agent header field does not commonly vary between requests and is sent to any server.  In that case, confirmation that a particular User-Agent value has been used provides little value.